htdocs/web_portal/GOCDB_monitor/check.php

```diff
@@ -17,7 +22,8 @@
 }
 
 if ($counts["error"] != 0) {
-    echo("An error has been detected while checking GOCDB services. Please check https://goc.egi.eu/portal/GOCDB_monitor/ to find out more\n");
+    echo("An error has been detected while checking GOCDB services. ".
+            "Please check https://goc.egi.eu/portal/GOCDB_monitor/ to find out more\n");
```

Comment:
The indentation feels weird to me, I'd probably prefer something like:

```suggestion
    echo(
        "An error has been detected while checking GOCDB services. ".
        "Please check https://goc.egi.eu/portal/GOCDB_monitor/ to find out more\n"
    );
```

But that's just the python programmer in me speaking, I'm looking into what if anything codeclimate and codacy are actually looking for style wise.

Reply:
As it is is closer to FIG-PSR12 (https://www.php-fig.org/psr/psr-12/) which is the default for vscode, although the second line indent is a bit weird. The whole conditional section auto-formats with psr12 according to vscode as -
```
if ($counts["error"] != 0) {
    echo ("An error has been detected while checking GOCDB services. " .
        "Please check https://goc.egi.eu/portal/GOCDB_monitor/ to find out more\n");
    exit(2); // return Nagios error code for CRITICAL
} else if ($counts["warn"] != 0) {
    echo ("At least one of GOCDB tests shows a warning. It is safe to ignore it anyway\n");
    exit(0); // we don't want notifications if there is just a warning
} else {
    echo ("All GOCDB tests are looking good\n");
    exit(0);
}
```
Your call, php or python?

Reply:
We should go for PHP style as it is PHP code. So would that be?:

```suggestion
    echo("An error has been detected while checking GOCDB services. ".
        "Please check https://goc.egi.eu/portal/GOCDB_monitor/ to find out more\n");
```

Vaguely related: I think through some wrangling we can get one of codeclimate/codacy to check specifically for things that go against PSR-12: Extended Coding Style. (Now captured here: https://github.com/GOCDB/gocdb/issues/270)